v0.8 (#1688)
system-ui/theme-ui: CHANGELOG.md

# Changelog

## v0.8.0 UNRELEASED

- **BREAKING:** `theme.config.useRootStyles` now defaults to `true` following
  deprecation of `useBodyStyles`. Styles from `theme.styles.root` are now added
  to `<html>` element by default.

- Moved theme configuration options to `config` object in theme.

  - **Deprecations:** `useRootStyles`, `useCustomProperties`,
    `useColorSchemeMediaQuery`, `useBorderBox`, and `useLocalStorage` options on

>     the theme object are now deprecated, and they will be removed in a future

Reviewer: ```suggestion
    the theme object are now scoped under a `config` object on the theme, and the root-level options, now deprecated, will be removed in a future
```
Sounds like we've gotten rid of those options until you get a little further—don't want to concern anyone

Author: But this is exactly what deprecated means, doesn't it? "It still works, it's just not in fashion anymore. Go use the new stuff." :P (Yes, your version is way better. Thanks, mate :D)

    release.

>     All these options are still supported, but they're grouped under
>     `theme.config` (e.g. `theme.config.useBorderBox`)

Reviewer: ```suggestion
   (e.g. you should be setting `theme.config.useBorderBox` instead of `theme.useBorderBox`)
```

  - ⚠ All config options should be migrated at once. If Theme UI sees
    `theme.config` exists, it won't look for options on the `theme`.

> - **Removed APIs deprecated in 0.6.0**

Reviewer: ```suggestion
- **Removed APIs, previously deprecated in 0.6.0**
```
